src/CalendarApp/CalendarApp.css

```diff
@@ -1,80 +1,124 @@
-
-
-
 .calendarTitle {
-    padding: 20px;
+    padding-bottom: 10px;
     color: grey;
-    font-size: 20px;
+    font-size: 18px;
     margin: 0;
+    text-align: start;
+    margin-left: 2%;
   }
 
-.containerOne {
+.calendarOne {
     display: flex;
     flex-direction: row;
     flex-wrap: wrap;
     align-items: flex-start;
     box-shadow: 0 0 8px silver;
-    margin: 20px 0;
-    padding: 10px;
+    margin: 5px 0;
+    padding: 5px;
   }
 
-.containerContent {
-      display: flex;
-      max-width: 100%;
-      flex-basis: 420px;
-      flex-direction: column;
-      flex-grow: 100;
-      align-items: stretch;
-      padding: 20% 1% 20% 1%;
-      
+.calendarContent {
+    display: flex;
+    max-width: 100%;
+    flex-basis: 420px;
+    flex-direction: column;
+    flex-grow: 100;
+    align-items: stretch;
+    padding: 1%;
     }
 
 .react-calendar__navigation__arrow {
-    font-size: 18px;
+    font-size: 16px;
     padding: 1px 15px;
     border: none;
+    color: rgb(90, 90, 250);
 }
 
 .react-calendar__navigation__label {
-    font-size: 18px;
+    font-size: 14px;
     font-weight: bold;
     border: none;
-    height: 40px;
+    height: 24px;
 }
 
 .react-calendar__tile {
```

Comment:
Suggest refrain from using component name in classNames, and instead add specificity to CSS file.
e.g. .react-calendar .navigation_label {
-- styles here --
}

Reply:
thanks Adam, this are specific to the react-calendar extension as discussed during the call today

Reply:
Ah sorry mate, I recall that now